### PR TITLE
TEST - Adds timeout to activate_data_product in preload R2

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

**Comment by lukecampbell** [comment]
@jamie-cyber1 ready for review and merge.

File: ion/processes/bootstrap/ion_loader.py [hunk 1]
```diff
@@ -3031,7 +3031,8 @@ def _load_DataProduct(self, row, do_bulk=False):
             self._register_id(row[COL_ID], res_id, res_obj)
 
             if not self.debug and get_typed_value(row['persist_data'], targettype="bool"):
-                dpms_client.activate_data_product_persistence(res_id, headers=headers)
```
Comment (jamie-cyber1) [inline ion/processes/bootstrap/ion_loader.py:3034]: Luke, I think the changes are equivalent?  If you don't pass in a timeout, it's in effect using the same timeout.

Comment (lukecampbell) [inline ion/processes/bootstrap/ion_loader.py:3034]: Your buildbot configuration should be setting this no?
In res/config/pyon.local.yml.

**Comment by lukecampbell** [comment]
I guess it's no longer needed.

**Pull request closed without merging**